tasks/post_to_database/orca_shared/shared_db.py

```diff
@@ -0,0 +1,184 @@
+"""
```

Comment:
Should I remove these shared files, requiring people to copy them in upon cloning the repo?